### PR TITLE
RMC admin PDA that scans

### PR DESCRIPTION
**Pull request by TsjipTsjip**
## About the PR

Adds an RMC admin PDA and adds it to default admin loadout. It scans with a RMC health analyzer UI!

## Why / Balance

Admins need to be able to tell time.

## Technical details

I am painfully aware of the utter madness that is adding the health scanner UI to all PDA's, but it's how upstream does it so who am I to argue?

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

n/a

## Breaking changes

n/a

**Changelog**

None.

### CODE REVIEW

**Comment by QuietlyWhisper** [comment]
- #3124 
maybe mix these two into one pr

**Comment by TsjipTsjip** [comment]
@QuietlyWhisper This PR now supercedes #3124 .

File: Resources/Prototypes/Entities/Objects/Devices/pda.yml [hunk 1]
```diff
@@ -98,6 +98,9 @@
         type: InstrumentBoundUserInterface
       enum.HealthAnalyzerUiKey.Key:
         type: HealthAnalyzerBoundUserInterface
+      # RMC-14: Add RMC health scanner UI key. (I'm just copying upstream okay... I know this is out of place. :( )
+      enum.HealthScannerUIKey.Key:
+        type: HealthScannerBoundUserInterface
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/Entities/Objects/Devices/pda.yml:103]: if you move this entire component onto the RMC pda you dont have to touch this file

File: Resources/Prototypes/_RMC14/Roles/Jobs/Fun/rmc_admin_ghost.yml [hunk 2]
```diff
@@ -11,9 +12,11 @@
   id: RMCMobAghostGear
   equipment:
     back: ClothingBackpackSatchelHolding
-    id: CMIDCardHighCommand # RMC-14: Changed from AdminPDA
+    id: RMCAdminPDA
   storage:
     back:
-    # RMC-14: Replaced loadout
-    - CMHealthAnalyzer
     - RMCAdminAccessConfigurator
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Roles/Jobs/Fun/rmc_admin_ghost.yml:18]: put the base default analyzer here as well for deeper damage debugging as well
Not sure if this is the right id
```suggestion
    - RMCAdminAccessConfigurator
    - HealthAnalyzer
```

File: Resources/Prototypes/_RMC14/Roles/Jobs/Fun/rmc_admin_ghost.yml [hunk 3]
```diff
@@ -3,6 +3,7 @@
   id: RMCAdminObserver
   noSpawn: true
   components:
+  - type: Marine
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Roles/Jobs/Fun/rmc_admin_ghost.yml:6]: smug said to remove this in the end for potential issues, we'll have a proper component for the purpose of this later
```suggestion
```

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_RMC14/Roles/Jobs/Fun/rmc_admin_ghost.yml [hunk 4]
```diff
@@ -11,9 +11,12 @@
...
   equipment:
     back: ClothingBackpackSatchelHolding
-    id: CMIDCardHighCommand # RMC-14: Changed from AdminPDA
+    id: RMCAdminPDA
   storage:
     back:
-    # RMC-14: Replaced loadout
-    - CMHealthAnalyzer
     - RMCAdminAccessConfigurator
+    - HandheldHealthAnalyzer
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Roles/Jobs/Fun/rmc_admin_ghost.yml:18]:
```suggestion
    - HandheldHealthAnalyzerUnpowered
```

**Review by QuietlyWhisper — APPROVED** [review]
thanks

**Pull request merged**